server/model/opening-hours.js

```diff
@@ -18,60 +18,59 @@ type OpeningHoursDay = {|
 export type OpeningHours = Array<OpeningHoursDay>;
 
 export const galleryOpeningHours: OpeningHours = [
-  {dayOfWeek: 'Monday',    closes: '00:00', note: 'Galleries closed'},
-  {dayOfWeek: 'Tuesday',   opens: '10:00', closes: '18:00'},
+  {dayOfWeek: 'Monday', closes: '00:00', note: 'Galleries closed'},
```

Comment:
If we want to keep this as-is, we can turn off the rule, either globally, or (perhaps better) case-by-case inline.